### ISSUE TITLE
High Sierra OSX : Black Orbitals 

### ISSUE THREAD

**Issue by cdsnow** [OpenChemistry/avogadrolibs#380]
**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: 1.2.0 (Git revision: c1fcc5b)
 - Qt: 4.8.7

**Desktop version: (please complete the following information):**
 - OS: MacOS
 - Version 10.13.6
 - Compiler (if you built from source).  From installer

**Describe the bug**

Lack of color in the orbital surfaces. I saw a lonely link to the apparent same issue in Mac OS Sierra on an old Avogadro bug site...

**To Reproduce**

I build a molecule (e.g. ethane), and then use Extensions => Orca => Generate Orca Input 
Then I add some print statements to the Orca script to inspect the orbitals. Note: the full script is reproduced below.
!  Normalprint 
%output 
Print[ P_Basis ] 2 
Print[ P_MOs ] 1 
end 

Back on the command line I executed the following:
icy:orca_demo csnow$ orca ethane.inp > ethane.out

From Avogadro I loaded the output in as follows:
File => Import => Molecule File => ethane.out

The very first time I ran the program, the resulting orbitals (e.g. HOMO, LUMO) looked normal: red and blue.  The next several attempts (reloading Avogadro) have resulted in black orbitals.

**Expected behavior**
Red and blue isosurfaces.

**Screenshots**

<img width="1291" alt="screen shot 2018-09-04 at 5 51 14 pm" src="https://user-images.githubusercontent.com/2815926/45063447-41feb180-b06b-11e8-8d75-8e20a0eba7cd.png">

**Additional context**
The full ORCA input script is:
~~~~
# avogadro generated ORCA input file 
# Basic Mode
# ethane
! RHF OPT def2-SVP 

!  Normalprint 
%output 
Print[ P_Basis ] 2 
Print[ P_MOs ] 1 
end 

* xyz 0 1
   C       -4.90043        3.94143       -0.00000
   C       -3.38837        3.94143       -0.00000
   H       -5.28480        2.91711        0.00787
   H       -5.28480        4.44677       -0.89101
   H       -5.28480        4.46039        0.88315
   H       -3.00400        3.42246       -0.88315
   H       -3.00400        3.43608        0.89101
   H       -3.00400        4.96574       -0.00787
*
~~~~

**Comment by ghutchis** [comment]
Yes, this is a known bug in Avogadro 1.2. Please consider trying Avogadro 2.0 beta:
https://www.openchemistry.org/downloads/

I can confirm that orbitals only render in red/blue.

You can *try* to re-set your Avogadro 1.x preferences by removing them. In the Finder, choose "Go to" and then ~/Library/Preferences

Find the file "net.sourceforge.Avogadro" and move it to the trash. When you restart, all the settings will revert to default, which should include red/blue.
